src/components/Popover/Popover.tsx

```diff
@@ -182,25 +181,28 @@ export const Popover = ({
 
   return (
     <FocusLock disabled={!activeFocus}>
-      <article
-        aria-describedby={ariaDescribedBy}
+      <div
         aria-hidden={!isActive}
-        aria-labelledby={ariaLabelledBy}
         className={componentClassName}
         onKeyDown={(e) => handleOnKeyDown(e)}
-        ref={ref}
-        role="dialog"
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
It might be fine to suppress the lint error, as long as the keyboard interaction is what we expect.

Is the idea of this component that the popover is sort of like a modal and you can tab onto the container?

Reply:
> It might be fine to suppress the lint error, as long as the keyboard interaction is what we expect.
> 
> Is the idea of this component that the popover is sort of like a modal and you can tab onto the container?

Yes I think so.. yes I think keep the suppress the lint error is makes sense. I have reversed `Popover.tsx` file back.